### ISSUE TITLE
Enhancement: Notification during bidding process

### ISSUE THREAD

**Issue by jseagrave21** [NewEconoLab/neo-ns#3]
*Note: this idea is based on my experience with testnet process*

A notification option by email or SMS during the bidding process would be very helpful, to alert a user if his /her current bid has been outbid.
This notification service could also alert the user to a change in stage of bidding (e.g. auction period to overtime bidding).

**Comment by Jim8y** [comment]
That will be a centralized service , not all NEO users will register their email address. Maybe NNS can provide email notification as optional.

Best regards,
Jinghui
________________________________

**Comment by jseagrave21** [comment]
Yes, exactly. I thought it would be a nice option. They could input it if desired, or leave blank if not. The email would not need to be stored on the blockchain. Instead it could be stored on the website server only.

**Comment by Jim8y** [comment]
Notification should rely on the wallet. Besides, I do agree with you, cause I'm gonna add WeChat notification for my WeChat mini program on NNS.
